specification/computeschedule/ComputeSchedule.Management/tspconfig.yaml

```diff
@@ -0,0 +1,11 @@
+emit:
+  - '@azure-tools/typespec-autorest'
+options:
+  '@azure-tools/typespec-autorest':
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/computeschedule.json"
+    examples-directory: "{project-root}/examples"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-resource-manager/all"
```

Comment:
per typespec linter, please change to extends:
"@azure-tools/typespec-azure-rulesets/resource-manager"